base/essentials.jl

```diff
@@ -165,6 +165,23 @@ function convert end
 
 convert(::Type{Any}, @nospecialize(x)) = x
 convert(::Type{T}, x::T) where {T} = x
+
+"""
+    Base.unwrap(x)
+
+Lower, or "unwrap" a value that has Base-defined representation. Currently
```

Comment:
What do "lower" and "Base-defined" mean exactly?

Reply:
Base-defined really means that you convert/transform your wrapper type to a value with a type that already has a `convert(T, x)` definition, which for the most part, will be Base/stdlib-defined types.

Reply:
OK. That doesn't sound very explicit to me, and if that's only try "for the most part" it shouldn't be presented as a general property. Why not just say "unwrap"?